install_config/configuring_sdn.adoc

```diff
@@ -199,19 +199,52 @@ This is only supported for {product-title} on Red Hat OpenStack Platform.
 ====
 endif::[]
 
-To enable *flannel* within your {product-title} cluster, set the following
-variables in your Ansible inventory file before running the installation.
+[IMPORTANT]
+====
+Neutron must be disabled, even when security groups are not being used.
```

Comment:
@bmeng The title of the section is "Neutron: port_security enabled", but it's not worded strongly enough to be one or the other. The fact that the BZ is about disabling Neutron port security made me word it that way. Might be better to word it like "configure Neutron port security to make it work" rather than one way or the other. @jliberma , if you can clarify, that would be great.